src/main/java/Main.java

```diff
@@ -0,0 +1,51 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Main {
+    private static final String INPUT = "sample-text-file2.txt";
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+
+    public static void main(String[] args) {
+        logger.warn("Application started");
+
+        Main main = new Main();
+        List<String> words = main.readFromFile(Path.of(INPUT));
+        main.showResult(words);
+
+        logger.warn("Application finished");
+    }
+
+    public List<String> readFromFile(Path path) {
+        logger.warn("Starting reading data");
+
+        List<String> stringList = new ArrayList<>();
+        try (BufferedReader bufferedReader = Files.newBufferedReader(Objects.requireNonNull(path))) {
+            while (bufferedReader.ready()) {
+                stringList.add(bufferedReader.readLine());
+            }
+        } catch (IOException e) {
+            logger.error("Can not read file from path " + path);
```

Comment:
add exception to logger to log stack trace

Reply:
Already done.

Reply:
first. only author of first comment accept fixes and mark resolve as resolved.
second. add exception to log mean add second parameter to logger method with throwable - logging system grab stacktrace autoamtically. - please correct this way
```suggestion
            logger.error("Can not read file from path " + path, e);
```

Reply:
1) Sorry for that.
2) Already done. Thank you for the hint.

Reply:
1. it's unnecessary - this is a part of CodeReview and learning process
good job